### DISCUSSION TITLE
supabase auth(google) is not working with nextjs 13

### DISCUSSION THREAD

**Discussion by MickyLaitkor** [supabase/supabase#16280] [Questions]
I tried to use  google auth with supabase. And working fine when trying to sign in and also authenticated on supabase but on frontend nextjs is not able to get session.  and I got some related issue and tried all the  solution  like downgrade the @supabase/gotrue-js  but still doesnot work.  how to fix this?

**Comment by 1shaked** [comment]
same, the code work in next12, but not on next 13 please create a sample repo

**Comment by BrunoFerreira95** [comment]
I am ready implement the google auth, in the nextjs 13, and it’s working.

Don’t forget to use ‘use client’ when you console.log (session), because need to run in the client version.

You also need to create cookies
